diagnostic.md

```diff
@@ -5,27 +5,32 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components. Explain why each piece might be it's own component.
 
     ```md
-    <!-- your response here -->
+    One example would be a hierarchy represented by a top level component called 'Recipe Title' and, below that visually, a
+    nested component called 'Ingredients'.  A change to one would not conceptually affect the other (unless that was the
+    desired outcome).  To see the 'Ingredient' component I might want to click on the 'Recipe Title'.  I could add different
+    ingredients without affectng the 'parent' component.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember generate component my-map
     ```
 
 1.  What files are created and/or edited to produce a component, and what are their responsibilities?
 
     ```md
-    <!-- your response here -->
+    There are two files generated.  component.js and template.hbs.  The component.js is responsible
+    for the actions and the template.hbs is responsible for the html and the page that is presented
+    to the User.
     ```
 
 1.  Suppose you have a component '`my-contact`', which is loaded from
     '`app/contacts/template.hbs`' when visiting the `/contacts` route. What is
     the syntax (code that is written) to render this component inside that template?
 
     ```html
-    <!-- your response here -->
+    {{#link-to 'my-contact' my-contact}} {{/link-to}}
```

Comment:
Because it's a component, we can't use `link-to`

```
{{#each model as |contact|}}
{{my-contact contact=contact}} <!-- most important part of response -->
{{/each}}
```